ebl/tests/fragmentarium/test_fragment_repository.py

```
@@ -105,6 +105,42 @@ def test_find_interesting(fragment_repository):
     ) == [interesting_fragment]
 
 
+def test_folio_pager_exception(fragment_repository):
+    query = "1841-07-26, 57"
+    with pytest.raises(NotFoundError):
+        list(fragment_repository.query_next_and_previous_fragment(query).values())
```

`list` and `.values()` are not needed. Making the call should raise the exception.